### ISSUE TITLE
UserHistory

### ISSUE THREAD

**Issue by Shimi4583** [razibchamp/FundArising-#30]
In user history,what will be included?Like details,Transaction date etc?

**Comment by razibchamp** [comment]
Yes. But not limited to.  Like any other history, it will contain every activity that has done by the user.

**Comment by razibchamp** [comment]
user history will be short descriptive activities whereas user report will be in tabular format

**Comment by Shimi4583** [comment]
What should I add now in userhistory now?
